cypress/e2e/08_add_monument/add_monument_workflow.cy.js

```diff
@@ -0,0 +1,631 @@
+describe('Going through the Add Monument Workflow', function () {
+
+    beforeEach(() => {
+        cy.login();
+        cy.visit('/plugins/init-workflow');
+    });
+
+    it('Go through the workflow and populate all fields', function () {
+        cy.contains('Workflows');
+        cy.contains('Add Monument').click();
+        cy.wait(2000);
+
+        // Add Monument 1st page
+        cy.get('[aria-label="Start new or please select from below"]').click();
+        cy.wait(2000);
+        cy.get('.select2-results__option').contains('HA/01').click();
+        cy.get('.btn-success').contains('Start New').click();
+        cy.wait(2000);
+
+        // Start tab
+        cy.wait(4000);
+        cy.get('.card_component.resourceid').contains('HA Number');
+        cy.get('.form-control').should('be.disabled');
+        cy.get('.btn-success').contains('Save and Continue').click();
+
+        // Heritage Asset Details tab
+        cy.contains('Site Name');
+        cy.get('[aria-label="Site Name"]').click().type('Testing');
+
+        cy.contains('Heritage Asset Type');
+        cy.get('[aria-label="Heritage Asset Type"]').click();
+        cy.wait(2000);
+        cy.get('.select2-results__option').first().click();
+
+        cy.contains('Historical Period').scrollIntoView();
+        cy.get('[aria-label="Historical Period"]').click();
+        cy.wait(2000);
+        cy.get('.select2-results__option').first().click();
+
+        // Condition Type
+        cy.contains('Condition Type');
+        cy.get('.card_component.condition_type').contains('Select an option').click();
+        cy.wait(2000);
+        cy.get('.select2-results__option').first().click();
+
+        // Category Type
+        cy.contains('Category Type');
+        cy.get('.card_component.category_type').contains('Select an option').click();
+        cy.wait(2000);
+        cy.get('.select2-results__option').first().click();
+
+        // Bibliographic Source
+        cy.get('.bibliographic_source_citation').contains('Bibliographic Source').scrollIntoView();
+        cy.wait(2000)
+         cy.get('.bibliographic_source_citation').contains('Add new Relationship').click();
+        cy.wait(2000);
+        cy.get('.select2-results__option').first().click();
```

Comment:
Again, this should've been added to the prerequisite tests similar to create person etc. Shouldn't use a random record as this wouldn't be present in CI